### DISCUSSION TITLE
auro-flight demo out of date?

### DISCUSSION THREAD

**Discussion by blackfalcon** [AlaskaAirlines/auro-flight#101] [Please help!] [answered]
@johnalaska messaged the team

> I had a meeting with Dale Sande yesterday where I tried to reproduce a bug I'd found in auro-flight (departure/arrival time font size changing at the wrong breakpoint) but it didn't happen for us. In fact the font size didn't change at all (this is the code that should do it, added oct 30: https://github.com/AlaskaAirlines/auro-flight/commit/bbfd5bfc8e4e89e15de41f58e5b058f92440bc23).
>
> I built auro-flight locally and ran the demo page and reproduced the bug.
Is there a manual step that needs to be done to bump the versions used on https://auro.alaskaair.com/components/auro/ ?
>
> FWIW my finding is that the fix needed for auro-flight is to update the dep on @aurodesignsystem/webcorestylesheets to 5.0.x
I'll create an issue before doing the work but I thought it best to ping the team before even doing that

Additionally, what brought up this conversation is that it was reported that the smaller size was defined as a bug by @erikalanghauser, but she was included in the issue that made this update https://github.com/AlaskaAirlines/auro-flight/issues/55#issuecomment-1703227291

There is clearly an issue with the proper version of this element either not appearing on the docsite or the feature is not working as expected on the docsite. 

But the real confusing issue is, the smaller text size appearing at breakpoints was a requested feature by @cwilliams7 and reviewed by @leeejune and @erikalanghauser. Is this feature no longer requested?

**Comment by erikalanghauser** [comment]
@blackfalcon - We still want the smaller text size at mobile breakpoint. The implementation just does not match the design spec. Weight is off on the times.

Design left, Implementation right:
![image](https://github.com/AlaskaAirlines/auro-flight/assets/34354487/f0f36e4d-1d51-4db3-a2f4-8f29933c041a)

It should use heading-300, for breakpoint-sm (320px-576px).

**Reply by erikalanghauser** [reply, marked as answer]
@blackfalcon - Also, currently sizes are changing but at **659px**, which is no longer an auro breakpoint.
So there are 2 issues:
* The sizes changing at a 659px breakpoint instead of 575px. (We have auro XS as 320px-575px)
* While a smaller size _does_ appear, it does not match the design spec exactly. (I did comment on the issue requesting the change, but was never asked to review implementation before it was released.)

**Reply by blackfalcon** [reply]
Ok, so what I misunderstood as the 'bug' was that it was even happening. But what you are saying is that we want the size change, but the behavior is not correct. That makes sense.

**Comment by blackfalcon** [comment]
What is also an issue is, why is this behavior not working as all on the Auro docs site? The following code should be using the latest bundled version of the element. 

```html
<script src="https://cdn.jsdelivr.net/npm/@aurodesignsystem/auro-flight@latest/dist/auro-flight__bundled.js" type="module"></script>
```

**Comment by blackfalcon** [comment]
New issues created

- https://github.com/AlaskaAirlines/auro-flight/issues/102
- https://github.com/AlaskaAirlines/auro-flight/issues/104

**Reply by blackfalcon** [reply]
@Patrick-Daly-AA these issues are ready for refinement and assignment.
